### DISCUSSION TITLE
Remember  / Restore connections on reopen on rustdesk

### DISCUSSION THREAD

**Discussion by jannoke** [rustdesk/rustdesk#9823] [General]
I do a lot of work on non-human machines. It means they live their own life and autoconnect ( passwords are remembered) is used. It would be good to have option to restore connections on next startup. This could be a tick on closing "reconnect next startup / always" or "restore previous connections" on next startup. Kinda like chrome or other browser works on restoring tabs on next startup. Instead I currently need to reconnect to around 15 machines manually to have quick monitoring on them.

**Comment by xlionjuan** [comment]
Create a batch script and connect them at once

**Comment by jannoke** [comment]
I guess that's the manual way to go. A lot of number typing later it works like charm.

**Reply by xlionjuan** [reply]
But you only need to do it once 👍
